golang/outbound.go

```diff
@@ -46,9 +46,13 @@ func (c *Connection) beginCall(ctx context.Context, serviceName string, callOpti
 		return nil, err
 	}
 
-	deadline, _ := ctx.Deadline()
+	deadline, ok := ctx.Deadline()
```

Comment:
consider defaulting a deadline ? In node we default you into 100ms if you do not set a timeout.